src/web/quick-start.md

```diff
@@ -0,0 +1,250 @@
+# Bevy on the Web
+
+Below are a set of instructions to run your Bevy project on the web, or in other words, in the browser. Others might say you want to compile for WebAssembly (often abbreviated to WASM). These are all the same thing.
+
+Note that these steps provide instructions to target WASM _only_. If you
+want your project to work on native (i.e. Windows, MacOS and/or Linux) _and_
+WASM, you will need to perform additional steps. See
+[Multi-Target](./web/multi-target.md).
+
+## Getting a WASM-compatible rendering back-end
+
+Bevy's rendering back-end currently does not support the web. You will need
+[bevy_webgl2](https://github.com/mrk-its/bevy_webgl2) instead. At the very
+least, you require the following crates and features in your `Cargo.toml`:
+
+```toml
+[dependencies]
+bevy = {version = "0.4", default-features = false, features = ["bevy_winit", "render"]}
+bevy_webgl2 = "0.4"
+```
+
+Note that some Bevy features (such as audio) are currently not compatible with
+WebAssembly and will produce compilation errors. Before adding features or 3rd
+party crates, please verify whether they are WASM-compatible.
+
+## Configuring the Bevy application builder
+
+You are now compiling a WASM-compatible rendering back-end, but you still need
+to load it in Bevy's application builder. In your code simply replace Bevy's
+`DefaultPlugins` with `bevy_webgl2::DefaultPlugins`:
+
+```rust
+App::build()
+    .add_plugins(bevy_webgl2::DefaultPlugins)
+```
+
+Alternatively add both Bevy's `DefaultPlugins` and `bevy_webgl2::WebGL2Plugin`:
+
+```rust
+App::build()
+    .add_plugins(DefaultPlugins)
+    .add_plugin(bevy_webgl2::WebGL2Plugin)
+```
+
+## Building for WebAssembly
+
+In order to practically build a Bevy project (or any Rust project) for the web,
+you need more than just a compiled binary. Simply targeting the
+`wasm32-unknown-unknown` platform and building is not sufficient. At the very
+least you also require a set of JavaScript mappings so the browser knows what
+functions in your binary to call. Depending on the complexity of your project
+this can get convoluted fast. Luckily, tooling exists to automate this.
+
+There are two common high-level methods that result in workable WASM binaries,
+JavaScript mappings and potentially other handy files:
+[wasm-pack](https://github.com/rustwasm/wasm-pack) and
+[cargo-make](https://github.com/sagiegurari/cargo-make).
```

Comment:
It might be clearer to organize this with two complete sets of instructions - one for wasm-pack and the other for cargo make rather than providing two sets of directions for each step.

Reply:
I think the best option here would be to split off only the `cargo-make` parts in their own section completely, and list the trade-offs of it vs `wasm-pack` there. This would slim down the quickstart part of the guide significantly, and also let us merge the multi-target bits in here as mentioned before (as the `cargo-make` template we use is not compatible with a multi-target approach without platform-specific features).